src/pages/components/overview/feature-flags.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Overview
+description:
+  Components are one of the key building blocks of the design system. Each
+  component has been designed and coded to solve a specific UI problem.
+tabs: ['Components', 'Accessibility status', 'Feature flags']
+---
+
+import { Launch } from '@carbon/icons-react';
+import { Link } from '@carbon/react';
+
+<PageDescription>
+
+Feature flags enable new behaviors and styling, allowing users to opt-in to new
+breaking changes while staying on the current major version.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Components with feature flags</AnchorLink>
+<AnchorLink>How to implement</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+When a new feature flag is introduced, it is set to false or "off" by default to
+ensure backward compatibility. We are not deprecating the current components but
+encourage all teams to use the new feature flag-enabled components in their
+products moving forward.
+
+## Components with feature flags
+
+The following components have feature flags, with changes related to either
+design or development.
+
+| Component                    | Design change                                                                    | Code change                                                                                                                                                                                  |
```

Comment:
Seeing this table live on the page, I kind of wish we went with the more [detailed table](https://www.figma.com/design/MDYwd15HhlWpnNZWhXYMKI/Feature-flag-implementation?node-id=875-68880&t=7Q8uLawLviGCK9xk-1) instead. I guess we can always discuss it more and update it later.

Reply:
I completely understand, and I also think discussing and updating it later makes sense, especially since this could evolve as other frameworks implement their feature flags as well.